docs/lib/project-setup/fragments/flutter/prereq/prereq.md

```diff
@@ -1,4 +1,4 @@
 - Install [Flutter](https://flutter.dev/docs/get-started/install) version 1.20.0 or higher
 - Setup your [IDE](https://flutter.dev/docs/get-started/editor?tab=androidstudio)
-- Android API level 16 (Jelly Bean) or higher
+- Android API level 21 (Android 5.0) or above
```

Comment:
This is typically known as Lollipop, I believe.

Reply:
@jpignata  I'm leaving it at Android 5.0 for now to align with the wording in other sections of our docs.